### PR TITLE
IRIS-4216 | Limit max value of unread notifications to 99

### PR DESCRIPTION
**Pull request by slayful**

### CODE REVIEW

File: extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_icons.scss [hunk 1]
```diff
@@ -1,12 +1,12 @@
 .wds-icon {
```
Comment (slayful) [inline extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_icons.scss:1]: This file was modified by other Design System changes.

File: extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_buttons.scss [hunk 2]
```diff
@@ -18,6 +18,8 @@
 	}
```
Comment (slayful) [inline extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_buttons.scss:18]: This file was modified by other Design System changes.

File: extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_avatar-stack.scss [hunk 3]
```diff
@@ -10,7 +10,7 @@
 	@for $i from 1 to 6 {
```
Comment (slayful) [inline extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_avatar-stack.scss:10]: This file was modified by other Design System changes.

File: extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/index.scss [hunk 4]
```diff
@@ -8,6 +8,7 @@
 @import 'menu-chevron';
```
Comment (slayful) [inline extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/index.scss:8]: This file was modified by other Design System changes.

File: extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index.scss [hunk 5]
```diff
@@ -1,6 +1,7 @@
 // Base variables
```
Comment (slayful) [inline extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index.scss:1]: This file was modified by other Design System changes.

File: extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.view.js [hunk 6]
```diff
@@ -200,13 +200,23 @@ define('ext.wikia.design-system.on-site-notifications.view', [
...
 				if (count > 0) {
 					this._$markAllAsReadButton.removeClass(isHiddenClass);
-					this._$notificationsCount.html(count).parent('.bubbles').addClass('show');
+					this._$notificationsCount.html(limitTo99(count))
+						.removeClass(isHiddenClass);
 				} else {
 					this._$markAllAsReadButton.addClass(isHiddenClass);
-					this._$notificationsCount.empty().parent('.bubbles').removeClass('show');
+					this._$notificationsCount.empty()
+						.addClass('wds-is-isHiddenClass');
```
Comment (kvas-damian) [inline extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.view.js]: Is it correct class name?

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**